src/Kernel/CompiledMethod.class.st

```diff
@@ -398,7 +404,7 @@ CompiledMethod >> displayStringOn: aStream [
 { #category : #accessing }
 CompiledMethod >> endPC [
 	"Answer the index of the last bytecode, as the trailer size varies we have to delegate"
```

Comment:
Maybe the comment should be updated since it’s not really délégation anymore? But this is a detail

Reply:
yes, I will fix it next